pkgs/tools/package-management/protonup/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, requests, configparser }:
+
+buildPythonPackage rec {
+  pname = "protonup";
+  version = "0.1.4";
+
```

Just to be on the save site.

```suggestion

  disabled = pythonOlder "3.6";
```

see https://github.com/AUNaseef/protonup/blob/main/setup.cfg#L19

---

error: undefined variable 'pythonOlder'

---

Oh, never mind. I forgot to specify it in the input parameters.